### PR TITLE
[IOTDB-1709][IOTDB-2316] Create module for iotdb v0.13 and add some new parameter for new feature

### PR DESCRIPTION
**Pull request by xuanronaldo**
1. create module for iotdb v0.13
1. repair template function for v0.13
2. support vector function for v0.13 (JDBC，Session)

### CODE REVIEW

File: iotdb-0.13/src/assembly/assembly.xml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+<assembly>
+    <id>iotdb-0.11</id>
```
Comment (stormbroken) [inline iotdb-0.13/src/assembly/assembly.xml:22]: fix id to iotdb-0.13

File: configuration/conf/config.properties [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 ################### 被测数据库配置 #######################
 # 被测试的数据库，目前的格式为{name}{-version}{-insert mode}(注意-号)其全部参考值参见README文件
-DB_SWITCH=IoTDB-012-SESSION_BY_TABLET
```
Comment (stormbroken) [inline configuration/conf/config.properties:7]: Because 0.13 is not stable now, so I think default value can be IoTDB-012-SESSION_BY_TABLET now

**Review by stormbroken — COMMENTED** [review]
Excellent Job!
Please fix some minor problems.

File: iotdb-0.13/src/assembly/assembly.xml (hunk 1 above)
Comment (xuanronaldo) [reply iotdb-0.13/src/assembly/assembly.xml:22]: got it

File: configuration/conf/config.properties (hunk 2 above)
Comment (xuanronaldo) [reply configuration/conf/config.properties:7]: got it

**Review by stormbroken — COMMENTED** [review]
LGTM!

**Review by stormbroken — APPROVED** [review]

**Pull request merged**